src/main/webapp/css/upload.css

```diff
@@ -50,6 +50,29 @@ small {
   font-size: 12px;
 }
 
+.ldBar {
```

Comment:
rename to load-bar?

Reply:
The ldBar class is required for the loading bar library.

https://loading.io/progress/